src/components/ProtestForm/ProtestForm.js

```diff
@@ -89,6 +90,7 @@ function ProtestForm({ initialCoords, submitCallback, defaultValues = {}, afterS
       try {
         params.coords = mapCenter;
         params.dateTimeList = dateTimeList;
+        params.streetAddress = streetAddressDefaultValue;
```

Comment:
What happens when creating a new protest.does this not override the value from the form?

Reply:
This was when I tried to find out the problem, I removed the line now